packages/web-runtime/src/pages/account.vue

```diff
@@ -157,14 +154,17 @@ export default defineComponent({
     const clientService = useClientService()
     const configurationManager = useConfigurationManager()
     const valuesList = ref<SettingsValue[]>()
+    const graphUser = ref()
```

Comment:
```suggestion
    const graphUser = ref<User>()
```

Reply:
will do later, as we don't have preferredLanguage in the user rn